apps/frontend/src/components/ApproveRedirect.svelte

```diff
@@ -5,14 +5,18 @@
   import AmountFormatter from '@baf-wallet/base-components/AmountFormatter.svelte';
   import { BafError } from '@baf-wallet/errors';
   import Spinner from 'svelte-spinner';
-
+  
```

Comment:
why is the frontend code being modified here? Did you make this branch off of `main` or `hope/formValidation`?

Reply:
hope/formValidation. Now I see that I should've used `main`